README.md

```diff
@@ -21,17 +21,18 @@ Also you need to configure sass loader, since all the styles are in sass format.
-| ------------------------ | ---------------- | -------------------------------------------------- | ---------------------------------------------- |
-| value                    | any              |                                                    | The initially selected value                   |
-| loadOptions              | function         | () => Promise.resolve({ options: [] })             | Function for fetching options for the combobox |
-| onSelect                 | function         | () => {}                                           | Selection callback function                    |
-| i18n                     | object           |                                                    |                                                |
-| i18n.getMessage          | function         | id => id                                           | Function for getting localized texts           |
-| modal                    | object           |                                                    |                                                |
-| modal.title              | string           | ''                                                 | Localized title of the modal                   |
-| modal.fields             | [string]         | []                                                 | List of fields to show as columns              |
-| modal.loadOptions        | function         | () => Promise.resolve({ data: [], totalCount: 0 }) | Function for fetching entries to the table     |
+| Prop name                | Type             | Default                                            | Description                                                           |
+| ------------------------ | ---------------- | -------------------------------------------------- | --------------------------------------------------------------------- |
+| value                    | any              |                                                    | The initially selected value                                          |
+| loadOptions              | function         | () => Promise.resolve({ options: [] })             | Function for fetching options for the combobox                        |
+| onSelect                 | function         | () => {}                                           | Selection callback function                                           |
+| mapTranslationKey        | function         | key => key                                         | Transforms the translation key before it is passed to i18n.getMessage |
+| i18n                     | object           |                                                    |                                                                       |
+| i18n.getMessage          | function         | id => id                                           | Function for getting localized texts                                  |
```

Comment:
i18n not valid anymore?

Reply:
Of course there was something to be forgotten 😄